packages/tldraw/api/api.json

```diff
@@ -14583,7 +14583,7 @@
               "text": "export interface TLUiContextMenuProps "
             }
           ],
-          "fileUrlPath": "packages/tldraw/src/lib/ui/components/ContextMenu.tsx",
+          "fileUrlPath": "packages/tldraw/.tsbuild-api/lib/ui/components/ContextMenu.d.ts",
```

Comment:
It wanted to do some api changes?

Reply:
it does the same on my machine, no idea why but i don't think we use these paths anywhere so it should be ok to let api-extractor be weird here.

Reply:
err looks like I was wrong and we use these paths in the docs so we should find a way to fix these .tsbuild-api paths. But that's obvs outside the scope of this PR so it's still fine to leave these as is.